### PR TITLE
Hello_world: Milestone 2

### PR DESCRIPTION
**Pull request by Donmark2k**
**In this Milestone, 2:**
1. created index.html file
2. created CSS file
3. styled h1 tag
4. changed the branch name from indexpage to created_indexpage
5. added title to pull request to

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Fundamental of HTML, CSS
+
```
Comment (jerryowusu) [inline README.md]: You are using the old readme template. :-1:  Kindly use the latest and new [readme template](https://github.com/microverseinc/readme-template) and make sure all header/sections are present. :1234:  You should also edit them to suit the project and your credentials :+1:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Donmark2k 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-there-reach


## Highlights
All linters are passing :heavy_check_mark: 
Put in a good effort :heavy_check_mark: 

## Required Changes ♻️

- [ ] Per the instructions, you are suppose to create .gitignore file and add the `test.md` and `node_modules` directory (as it can be added by accident while setting up linters)
to it. When this is done properly the `test.md` file will not appear in your github repository. :100: Kindly check and update your files accordingly :pray: 

_Check the comments under the review._

## Optional suggestions

- [ ] I can see in this repository that you have already presented another pull request which has already been reviewed by another reviewer. You do not need to present a different pull request for each review. :+1: So kindly make the changes and present this same pull request next time (please do not create another pull request for the next review :pray: ). You can create a different pull request when you are working on a big project with multiple milestone when one has been approved and merged :wink: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Donmark2k) [reply README.md]: I have resolved it though the first reviewer marked it as Optional

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```
Comment (roshan-bajgain) [inline .gitignore]: - [ ] As requested by previous reviewers you should put the  `test.md file` in `.gitignore` file and It should not present on your repo. Please fix it as I see the test.md file in your repo.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Donmark2k, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (Donmark2k) [reply .gitignore]: Please if there is another way to put test.md file in .gitignore file ,please tell.

I already put the test.md in the .gitignore file. What else do you want me to do?

Comment (roshan-bajgain) [reply .gitignore]: @Donmark2k, Please try to push the code and check whether there is test.md file present on the repo or not.

Comment (roshan-bajgain) [reply .gitignore]: @Donmark2k, normally it happens when you already upload files, and after 
  that you add your .gitignore file.  please try this to fix issue:
`git rm -rf --cached .`
`git add .` 
This removes all files from the repository and adds them back

**Review by muneebulrehman — APPROVED** [review]
Hi @Donmark2k , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good follow-up with the previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**